decidim-consultations/db/migrate/20180115132000_add_end_voting_date_to_decidim_consultations.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class AddEndVotingDateToDecidimConsultations < ActiveRecord::Migration[5.1]
+  def change
+    add_column :decidim_consultations, :end_voting_date, :date
+
+    Decidim::Consultation.find_each do |consultation|
```

Comment:
Same thing here as in https://github.com/decidim/decidim/pull/3106/files#r177486262.

Reply:
I'll check @deivid-rodriguez